CRM/Utils/Money.php

```diff
@@ -74,43 +69,55 @@ public static function format($amount, $currency = NULL, $format = NULL, $onlyNu
       $currency = $config->defaultCurrency;
     }
 
+    $currency = self::getCurrency($currency);
+    $moneyAmount = Money::of($amount, $currency, NULL, RoundingMode::CEILING);
+
     if ($onlyNumber) {
-      $amount = self::formatLocaleNumericRoundedByCurrency($amount, $currency);
-      return $amount;
+      $formatter = new \NumberFormatter(CRM_Core_I18n::getLocale(), \NumberFormatter::CURRENCY);
+      $formatter->setSymbol(\NumberFormatter::CURRENCY_SYMBOL, '');
+      $formatter->setSymbol(\NumberFormatter::MONETARY_GROUPING_SEPARATOR_SYMBOL, $config->monetaryThousandSeparator);
```

Comment:
@mattwire hopefully these lines become history but also not sure how much this relates to disabled membership types on the contact tab

Reply:
@seamuslee001 Oh sorry I've done it again. Those commits should not have been in this PR - now updated.